.flask_version/run.py

```diff
@@ -0,0 +1,22 @@
+# This is a quick and dirty code!
+from flask import Flask
+from flask import request
+from sample_code.endpoints import endpoint
+
+app = Flask(__name__)
+
+
+@app.route("/")
+def hello():
+    return "Hello World!"
+
+
+@app.route("/api/v1/test/entry", methods=["POST"])
+def test_entry():
```

Comment:
What about passing `body` instead?

Reply:
Seems not possible: https://stackoverflow.com/questions/10434599/get-the-data-received-in-a-flask-request